splunktaucclib/cim_actions.py

```diff
@@ -427,7 +427,7 @@ def result2stash(
                     if mapexp(real_key):
                         _raw += ', {}="{}"'.format("orig_" + real_key.lstrip("_"), val)
```

Comment:
no f-string here?

Reply:
That's question to `pyupgrade`, I think the tool was confused, because it will be a big and complex f-string :)